integration-tests/custom-networks.test.ts

```diff
@@ -0,0 +1,55 @@
+import {createWallet, IWallet} from '@docknetwork/wallet-sdk-core/src/wallet';
+
+describe('Custom networks', () => {
+  let wallet: IWallet;
+  const networks = [
+    {
+      name: 'polygon',
+      id: 'polygon',
+      configs: '{}',
+    },
+    {
+      name: 'Polygon mumbai',
+      id: 'mumbai',
+      configs: '{}',
+    },
+  ];
+
+  it('should create a wallet with custom networks', async () => {
+    wallet = await createWallet({
+      databasePath: ':memory:',
+      networks,
+    });
+  });
+
+  it('should add documents to polygon network', async () => {
+    await wallet.addDocument({
```

Comment:
how do we know the document was added? This proves the call didn't blow up, but it doesn't assert that the doc was stored properly

Reply:
I've updated the test